EEBOS/SCROWL
# [sc-787] - Chore/Editor framing

### Short summary
The editor framing consists of sidebars (left, right) and top tabs components that surround the editor.

More details about the interface can be found in [this Notion doc](https://www.notion.so/Course-Editor-b3bac69233694311a99c6eae129a77f7).

The final designs have not been provided yet, so this PR tackles the functionality only, little styles were added in order to position the elements. The links, buttons, and sections will be styled once we get the final designs.

---

### Test steps

1. On the home page click on `Create New` to go to the editor page.
2. Use the top links to load the `Build` and `Structure` pages.
3. Ensure the `Home` link takes you to the home page.
